### DISCUSSION TITLE
[WASM] In need of a solution where custom attached properties work in XAML

### DISCUSSION THREAD

**Discussion by IneedHelp** [unoplatform/uno#15823] [Q&A]
Hello!
Can anybody help me with providing a sample project/solution where custom attached properties are working fine when used in XAML?
Apparently this should be something that's working, but in my case they just get ignored and if someone could provide a sample project where they do work, it could help me better understand what the issue might be.
Thank you in advance!

**Comment by jeromelaban** [comment]
Related to https://github.com/unoplatform/uno/issues/15813.

**Comment by IneedHelp** [comment]
Right, not needed anymore since I learned what the cause of the issue was.
